code/game/objects/items/stacks/medical.dm

```diff
@@ -95,37 +106,44 @@
 		M.heal_bodypart_damage((heal_brute/2))
 		return TRUE
 	if(iscarbon(M))
-		return heal_carbon(M, user, heal_brute, 0)
-	to_chat(user, "<span class='warning'>You can't heal [M] with the \the [src]!</span>")
+		return heal_carbon(M, user, heal_brute, heal_burn)
+	to_chat(user, "<span class='warning'>You can't heal [M] with \the [src]!</span>")
 
 /obj/item/stack/medical/bruise_pack/suicide_act(mob/user)
 	user.visible_message("<span class='suicide'>[user] is bludgeoning [user.p_them()]self with [src]! It looks like [user.p_theyre()] trying to commit suicide!</span>")
 	return (BRUTELOSS)
 
 /obj/item/stack/medical/gauze
 	name = "medical gauze"
-	desc = "A roll of elastic cloth that is extremely effective at stopping bleeding, but does not heal wounds."
+	desc = "A roll of elastic cloth, perfect for stabilizing all kinds of wounds, from cuts and burns, to broken bones. "
 	gender = PLURAL
 	singular_name = "medical gauze"
 	icon_state = "gauze"
-	var/stop_bleeding = 1800
-	self_delay = 20
-	max_amount = 12
+	self_delay = 50
+	other_delay = 20
+	max_amount = 8
+	amount = 4
 	grind_results = list(/datum/reagent/cellulose = 2)
 	custom_price = 100
+	absorption_rate = 0.25
+	absorption_capacity = 5
+	splint_factor = 0.25
+
+// gauze is only relevant for wounds, which are handled in the wounds themselves
+/obj/item/stack/medical/gauze/try_heal(mob/living/M, mob/user, silent)
+	var/obj/item/bodypart/limb = M.get_bodypart(check_zone(user.zone_selected))
+	if(limb)
+		if(limb.brute_dam > 40)
+			to_chat(user, "<span class='warning'>The bleeding on [user==M ? "your" : "[M]'s"] [limb.name] is from bruising, and cannot be treated with [src]!</span>")
+		else
+			to_chat(user, "<span class='warning'>There's no bleeding on [user==M ? "your" : "[M]'s"] [limb.name]</span>")
+
+/obj/item/stack/medical/gauze/eight
+	amount = 8
 
 /obj/item/stack/medical/gauze/twelve
 	amount = 12

```

Comment:
change this value to 8 so that it's not over max_amount, and maybe add a comment about it too